lib/rspec/rails/configuration.rb

```diff
@@ -80,7 +81,8 @@ def self.initialize_configuration(config) # rubocop:disable Metrics/MethodLength
       config.include RSpec::Rails::FixtureSupport
 
       if ::Rails::VERSION::STRING > '5'
-        config.add_setting :file_fixture_path, default: 'spec/fixtures/files'
+        config.singleton_class.class_eval { attr_accessor :file_fixture_path }
```

Comment:
There's `singleton_class` in Ruby 2.2.2, which is minimum requirement of Rails 5. But `attr_accessor` is still private there.